fe-nc-news/src/Comments.jsx

```diff
@@ -0,0 +1,35 @@
+import getComments from './utils/GetComments'
+import {React, useState, useEffect} from 'react'
+
+const Comments = ({article_id}) => {
+ const [comments, setComments] = useState([])
+ useEffect(()=>{
+    getComments(article_id).then((commentData)=>{
+        setComments(commentData)
+    })
+    }, [article_id])
+return (
+<>
+<h3>Comments</h3>
+    <div className="commentGrid">
+    {comments.length > 0 ?(
+    
+    comments.map((comment)=>{
+        return (
+            <div className="commentCard" key={comment.comment_id}>
```

Comment:
I'm seeing a lot of defaulting to using a <div> throughout this code. Its not providing the accessibility it could be. [Here's a short article](https://dev.to/kenbellows/stop-using-so-many-divs-an-intro-to-semantic-html-3i9i) for you to take a look at.